TKeazirian.HTTPServer/Response/HttpMethods.cs

```diff
@@ -0,0 +1,11 @@
+namespace TKeazirian.HTTPServer.Response;
+
+public static class HttpMethods
+{
+    public static readonly string Get = "GET";
+    public static readonly string Post = "POST";
+    public static readonly string Options = "OPTIONS";
+    public static readonly string Head = "HEAD";
+    public static readonly string Put = "PUT";
+    public static readonly string Patch = "PATCH";
```

Comment:
this is a begining of a great idea. I see `string` and `List<string>` all throughout the code. Is it time to introduce a Type?  This way, instead of passing strings around, you can use just these methods?